utils/avro-components-utils/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/SchemaWithMetadata.scala

```diff
@@ -14,7 +15,8 @@ case class SchemaWithMetadata(schema: ParsedSchema, id: Int)
 object SchemaWithMetadata{
   def apply(schemaMetadata: SchemaMetadata): SchemaWithMetadata = {
     schemaMetadata.getSchemaType match {
-      case "AVRO" => SchemaWithMetadata(new AvroSchema(schemaMetadata.getSchema), schemaMetadata.getId)
+      case AvroSchema.TYPE => SchemaWithMetadata(new AvroSchema(schemaMetadata.getSchema), schemaMetadata.getId)
```

Comment:
I wonder if we can do it somehow "automatically"... e.g. using SchemaRegistryClient.parseSchema?

Reply:
Yeah, we had a conversation about it with @gadomsky - Confluent do exactly the same as we do - they even have their own `nonRestrictiveParseSchema` `(validateDefaults = false)`